src/fft.jl

```diff
@@ -176,6 +176,14 @@ end
     ccall((:fftwf_plan_with_nthreads,libfftw3f), Cvoid, (Int32,), nthreads)
 end
 
+function get_num_threads()
+    @static if fftw_vendor == :fftw
+        ccall((:fftw_planner_nthreads,libfftw3), Cint, ())
+    else
+        error("This function is not currently supported by mkl")
```

Comment:
I'm worried about this, especially if we start using `get_num_threads` more extensively (e.g. to accept an `nthreads` keyword in planning).   Maybe for MKL we should implement the fallback solution of keeping our own global nthreads variable mirroring the one in FFTW/MKL.

Reply:
That's a good idea. I could just use similar code as what was in my first PR. I just had a global that cached the last number of threads.